workflow/scripts/generate_volume_qc.py

```diff
@@ -0,0 +1,38 @@
+import json
+import shutil
+from pathlib import Path
+from distutils.dir_util import copy_tree
+from zarrnii import ZarrNii
+import math
+
+# directory containing the volume rendering files
+resource_dir = Path(snakemake.output.resources)
+# where html file should be written
+html_dest = snakemake.output.html
+
+# inputted ome-zarr path
+ome_data = snakemake.input.ome
+
+# move volume renderer into the subjects directory
+copy_tree("qc/resources/volViewer", str(resource_dir))
+shutil.move(resource_dir / "volRender.html", html_dest)
+
+# Get most downsampled ome-zarr image
+ds_z = ZarrNii.from_path(ome_data,level=5, channels=[0,1])
```

Comment:
hard-coding the channels here fails when we have a different number of channels, but I think the best solution is to make a fix in zarrnii so that the default channels selects all of them (instead of just channel 0).. I'll make a note there to fix it, then we can change this line in a future PR..